devito/codeprinter.py

```diff
@@ -4,26 +4,31 @@
 
 
 class CodePrinter(CCodePrinter):
-    """
-    CodePrinter extends sympy.printing.ccode.CCodePrinter
+    """Decorator for sympy.printing.ccode.CCodePrinter.
+
+    :param settings: a dictionary containing relevant settings
     """
     def __init__(self, settings={}):
         CCodePrinter.__init__(self, settings)
 
     def _print_Indexed(self, expr):
-        """
-        override method in CCodePrinter
+        """override method in CCodePrinter
```

Comment:
This first line is not particularly informative. Also can we always start a docstring with caps to be consistent.